cmd/world/root/root_test.go

```diff
@@ -115,8 +114,17 @@ func TestCreateStartStopPurge(t *testing.T) {
 	err = createCmd.Execute()
 	assert.NilError(t, err)
 
+	defer func() {
+		// Purge cardinal
+		rootCmd.SetArgs([]string{"cardinal", "purge"})
+		err = rootCmd.Execute()
+		assert.NilError(t, err)
+
+		// Remove dir
+		os.RemoveAll(gameDir)
```

Comment:
This should be left in its own `defer os.RemoveAll(gameDir)` line just after line 107. We want to delete the temp directory even if the assert on line 115 fails.

Reply:
updated